README.md

```diff
@@ -87,27 +81,17 @@ git clone https://github.com/acl2/acl2 .
 
 ### Contributing
 
-See the documentation for [how to contribute][git tips].
-
-Even though we have merged the Community Books (formerly acl2-books) and
-ACL2 System (formerly acl2-devel) repositories into one, changes should
-be made only to the `books/` subdirectory unless you are Matt Kaufmann
-or J Moore, since everything outside `books/` is part of the ACL2
-system.  (If you have suggestions for system changes, they should be
-emailed to [Matt or J](mailto:kaufmann@cs.utexas.edu), as has been done
-in the past.)  An [automated testing mechanism][jenkins] vets user
-contributions.
+See the documentation for [how to contribute][how-to-contribute].
 
-[git tips]: http://www.cs.utexas.edu/users/moore/acl2/manuals/latest/index.html?topic=ACL2____GIT-QUICK-START
-[jenkins]: ./books/build/jenkins/README.md
+[how-to-contribute]: http://www.cs.utexas.edu/users/moore/acl2/manuals/latest/index.html?topic=ACL2____HOW-TO-CONTRIBUTE
 
 ### Staying Informed
 
-We invite anyone who is using this repository to join the [acl2-books
-mailing list][acl2-books], which receives commit messages and other
-discussion related to ACL2 system- and book-related development.
+We encourage users of ACL2 to join our [mailing lists][mailing-lists]
+and take advantage of other [ACL2 Community resources][community].
```

Comment:
NIT: Are we capitalizing "community" in "ACL2 community"? There is an instance in earlier in the README where it is not capitalized. But the short in `:doc community` has it capitalized.

I guess there are other similar inconsistencies in the README (not related to your changes here). There is one instance where "system" in "ACL2 System" is not capitalized, in contrast to the rest of the text. Also uncapitalized "community books" and "ACL2 home page" links.

Reply:
I haven't paid much attention to capitalization so far, but it would be good to be consistent.  Maybe we can reach consensus by having a discussion on Zulip.

Reply:
It doesn't have to hold up this PR; we can always come back to it later.